.travis.yml

```diff
@@ -5,3 +5,13 @@ after_success:
 - npm run coveralls
 install:
 - npm install
+before_deploy: npm run build
+deploy:
+  provider: bintray
+  file: "bintray.json"
+  user: "larrystone"
```

Comment:
Have you looked at these?
https://github.com/openmrs/openmrs-owa-addonmanager/blob/master/.travis.yml
https://github.com/openmrs/openmrs-owa-cohortbuilder/blob/master/.travis.yml
https://github.com/openmrs/openmrs-owa-obsadmin/blob/master/.travis.yml